src/ontology/mondo-edit.obo

```diff
@@ -445047,6 +445048,14 @@ xref: UMLS:C2718017 {source="MONDO:equivalentTo"}
 is_a: MONDO:0021179 {source="MESH:D057177", source="UMLS:C2718017"} ! proteostasis deficiencies
 created_by: http://orcid.org/0000-0002-4142-7153
 
+[Term]
+id: MONDO:0700040
+name: neonatal jaundice due to ABO incompatibility
+def: "Jaundice that appears during the neonatal period due to high levels of unconjugated bilirubin that are a result of maternal-fetal ABO incompatibility." [https://orcid.org/0000-0002-4142-7153, PMID:20598320, PMID:24382531]
+is_a: MONDO:0002901 {source="https://orcid.org/0000-0002-4142-7153"} ! blood group incompatibility
```

Comment:
I don't think this is the best parent, as it is due to ABO incompatibility but it is not a hematologic disorder.

Also, blood group incompatibility has a comment: May be obsoleted as it represents a finding

I'm not totally sure what the best parent is, maybe metabolic disease (following SNOMED: https://www.ebi.ac.uk/ols/ontologies/snomed/terms?iri=http%3A%2F%2Fsnomed.info%2Fid%2F16716281000119108)

I added an xref to SNOMED: 16716281000119108

Reply:
looking at the ontology, I updated the parent to 'hyperbilirubinemia', which is itself a 'bilirubin metabolism disease'